srcdeps-maven-quickstarts/srcdeps-mvn-git-profile-and-properties-quickstart/jar/src/main/java/org/l2x6/srcdeps/quickstarts/mvn/git/prop/profile/GreetingConsumer.java

```diff
@@ -14,10 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import org.codehaus.plexus.util.FileUtils;
+package org.l2x6.srcdeps.quickstarts.mvn.git.prop.profile;
+
+public class GreetingConsumer {
+
+    public String retrieveGreeting() {
+        return new org.l2x6.maven.srcdeps.itest.api.generated.GeneratedGreeter().getGreeting();
```

Comment:
No matter - there is only one occurrence. It came due to the fact that I did not have `GeneratedGreeter` in the class path of the project and I had to write it all by hand.